src/CMakeLists.txt

```diff
@@ -131,13 +132,20 @@ if(BUILD_WITH_TESTS)
 	enable_testing()
 endif()
 
+# If these are not explicitly set to empty, aws-sdk-cpp will take 
```

Comment:
These variables are no longer needed since we are building aws sdk separately now.

Reply:
Removed